src/r2mm/manager/PackageDexieStore.ts

```diff
@@ -174,3 +184,24 @@ export async function isLatestPackageListIndex(community: string, hash: string)
 export async function setLatestPackageListIndex(community: string, hash: string) {
     await db.indexHashes.put({community, hash, date_updated: new Date()});
 }
+
+async function getDexiePackage(community: string, packageName: string) {
+    const pkg = await db.packages.where({community, full_name: packageName}).first();
+
+    if (!pkg) {
+        throw new Error(`Couldn't find package "${packageName}" in ${community} packages`);
+    }
+
+    return pkg;
+}
+
+async function getDexieVersion(community: string, packageName: string, versionNumber: string) {
```

Comment:
Similar for here except the problem is worse. This makes it sound like you'd be getting the currently installed version of the Dexie library.

Reply:
Renamed [the function](https://github.com/ebkr/r2modmanPlus/pull/1485/files#diff-2122463b05e5c9342979622ca58d48970d706008d1d9df423063e69e07055988R200).